_posts/2020-12-02-sf_ismir20.md

```diff
@@ -0,0 +1,61 @@
+date: 2020-12-02
+author: Ruchit Agrawal
+author-id: ruchit
+background: /posts/sf_ismir20/sf.jpeg
+--- 
+This year's ISMIR was held virtually, following the trend of online conferences smitten by the COVID-bug.  I attended [ISMIR 2020](https://www.ismir2020.net/), the 21<sup>st</sup> International Society for Music Information Retrieval Conference, from the 11<sup>th</sup> to the 16<sup>th</sup> of October, 2020. I was delighted to find fascinating papers related to my current research. This blog post summarizes one such paper that piqued my interest. The paper is titled [“Learning to read and Follow music in Complete score sheet images”](https://arxiv.org/abs/2007.10736); and the authors are Florian Henkel, Rainer Kelz, and Gerhard Widmer, from JKU Linz, Austria. 
+
+## Score following in raw sheet images ##
+
+The work presented in this paper aims at score following in sheet music that is given as unprocessed images. The general goal of score following is to determine the current position in the score for an incoming music performance. The authors build upon their previous work presented in ISMIR 2019[^1] and extend the capabilities of the score following model. The novelty of this work lies in the fact that they deal directly with complete raw images of sheet music (although this is done one page at a time).
+Existing methods for score following rely on either computer-readable representations of the score (such as MIDI or MusicXML) or work with small snippets of the sheet images. While the former representation is not readily available and/or depends on robust OMR, the latter is unable to capture context and disambiguate between different possible matching positions. It also has another constraint that the sheet image excerpt has to correspond partially to the incoming audio excerpt, in the absence of which the tracker is lost and subsequent predictions are incorrect. Additionally, the methods that work with snippets of audio and sheet image need an “unrolled” score representation, i.e. the staves have to be cut out from the sheet image and stitched together in a sequence. 
+
+The authors attempt to overcome this limitation by working with entire pages of sheet music images. They extend the work initially proposed in [^1] to be able to handle the temporal aspect of score following, i.e. disambiguate multiple positions in the sheet image that match with the audio excerpt. 
+
+## A brief detour: Referring image segmentation
+The basic idea is motivated by “referring image segmentation” (cite), a task in computer vision that aims to identify a region in an image, given a language expression as the query. To better understand this task, take a look at this figure.
+<figure class="figure w-100">
+  <img src="{{ '/posts/sf_ismir20/referring.png' | relative_url }}" alt="Referring image segmentation" class="figure-img img-fluid mx-auto d-flex">
+  <figcaption class="figure-caption text-center">
+  Referring image segmentation. Figure from [^3]
```

Comment:
The citation [^3] doesn't work in the caption. I'm not sure if it's possible to fix it.